install/kubernetes/cilium/templates/hubble/peer-service.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if and .Values.hubble.enabled .Values.hubble.listenAddress .Values.hubble.peerService.enabled }}
```

Comment:
@nathanjsweet I've left this file in templates/hubble like in the upstream commit, although it's at odd with the v1.10 layout.